app/addons/documents/index-results/apis/base.js

```diff
@@ -24,12 +25,13 @@ export const resetState = () => {
   };
 };
 
-export const newResultsAvailable = (docs, params, canShowNext) => {
+export const newResultsAvailable = (docs, params, canShowNext, docType) => {
```

Comment:
I've been wondering about this. I don't think we should have any `action` functions in the api. The `api` files should just do ajax queries and be a basic api to interact with CouchDB. I don't think the `api's` should know about redux or anything like that. They should just pass data and then the action functions in the `action.js` files should do any required dispatching. What do you think?